client/clientservice/src/request_service.cpp

```diff
@@ -67,6 +67,11 @@ void RequestServiceCallData::populateResult(grpc::Status status) {
 void RequestServiceCallData::sendToConcordClient() {
   bft::client::Msg msg;
   if (request_.has_typed_request()) {
+    google::protobuf::Any app_request = request_.typed_request();
+    size_t request_size = app_request.ByteSizeLong();
+    std::string request(request_size, '\0');
+    app_request.SerializeToArray(request.data(), request_size);
+    msg = bft::client::Msg(request.begin(), request.end());
     std::string str = request_.typed_request().SerializeAsString();
     msg = bft::client::Msg(str.begin(), str.end());
```

Comment:
Hi Celine, why is `msg` being assigned twice?
(lines 74 and 76)

Reply:
Thanks Teo. Those lines are to be removed. Merging error. I have updated changes